react/README.md

```diff
@@ -51,7 +51,7 @@ You can also build the image yourself from the /react directory:
 
 ```bash
 ./dev.sh build
-OPEN_API_KEY=${OPENAI_API_KEY} ./dev.sh ../path/to/any/frontend/repo/on/machine
+OPENAI_API_KEY=${OPENAI_API_KEY} ./dev.sh ../path/to/any/frontend/repo/on/machine
```

Comment:
+1 to fix typo, but `OPENAI_API_KEY=${OPENAI_API_KEY}` - is same as `a=a` in python. What about:
```
OPENAI_API_KEY=your_api_key
./dev.sh ../path/to/any/frontend/repo/on/machine
```
or
```
OPENAI_API_KEY=your_api_key ./dev.sh ../path/to/any/frontend/repo/on/machine
```